templates/workspaces/base/terraform/storage.tf

```diff
@@ -20,6 +21,12 @@ resource "azurerm_storage_share" "shared_storage" {
   ]
 }
 
+resource "azurerm_storage_container" "stgcontainer" {
+  name                  = "default"
```

Comment:
default?
can't this have a more meaningful name?

Reply:
how about `datalake`?